core/src/impl/KokkosExp_IterateTileGPU.hpp

```diff
@@ -875,7 +875,7 @@ struct is_array_type<T[]> : std::true_type {
 template <typename T>
 using value_type_storage_t =
     typename std::conditional_t<is_array_type<T>::value, std::decay<T>,
-                                std::add_lvalue_reference<T> >::type;
+                                std::add_lvalue_reference<T>>::type;
```

Comment:
What do you think of the following ?

```cpp
template <typename T>
using value_type_storage_t =
    std::conditional_t<is_array_type<T>::value, std::decay_t<T>,
                                std::add_lvalue_reference_t<T>>;
```

Reply:
Sure, see https://github.com/kokkos/kokkos/pull/7539/commits/2980f51adb93388e4feb02567273b884b42ce320.